common/remote/sockfile.cpp

```diff
@@ -6376,9 +6378,11 @@ class CRemoteFileServer : implements IRemoteFileServer, public CInterface
             }
             else
             {
+                responseWriter->outputBeginArray("dfs:Row");
```

Comment:
@jakesmith technically "dfs:Row" is not correct.  The contents of Row are defined by the ECL Record specific schema, not by the schema identified by the unique id "urn:hpcc:dfs".  I would only add the dfs namepace to the EOG boolean.